tests/integration/dom0-template/var/lib/net-snmp/snmpd.conf

```diff
@@ -0,0 +1,55 @@
+#
+# net-snmp (or ucd-snmp) persistent data file.
+#
+############################################################################
+# STOP STOP STOP STOP STOP STOP STOP STOP STOP 
+#
+#          **** DO NOT EDIT THIS FILE ****
+#
+# STOP STOP STOP STOP STOP STOP STOP STOP STOP 
```

Comment:
```suggestion
# STOP STOP STOP STOP STOP STOP STOP STOP STOP
#
#          **** DO NOT EDIT THIS FILE ****
#
# STOP STOP STOP STOP STOP STOP STOP STOP STOP
```
Remove trailing spaces (can be comitted by clicking the button "commit suggestion" below/above)